src/styles/editor.module.css

```diff
@@ -207,6 +207,18 @@
     background: #eee;
 }
 
+@keyframes flash {
```

Comment:
This didn't draw my attention to the button appearing, it could be made less subtle!

Reply:
To be honest, I can't think of how to improve this without it being overly obnoxious. I'm sure once the content team know it's there they won't need it to be highlighted, so maybe this flash ends up being more cosmetic?